packages/pipeline/test/parsers/token_metadata/index_test.ts

```diff
@@ -0,0 +1,51 @@
+import * as chai from 'chai';
+import 'mocha';
+
+import { MetamaskTrustedTokenMeta, ZeroExTrustedTokenMeta } from '../../../src/data_sources/trusted_tokens';
+import { TokenMetadata } from '../../../src/entities';
+import { parseMetamaskTrustedTokens, parseZeroExTrustedTokens } from '../../../src/parsers/token_metadata';
+import { chaiSetup } from '../../utils/chai_setup';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+const ZEROEX_CONTRACT_ADDRESS = '0xE41d2489571d322189246DaFA5ebDe1F4699F498';
+
+const ZRX_DECIMALS = 18;
+
+const metamaskTrustedToken: MetamaskTrustedTokenMeta = {
```

Comment:
I think we should use actual API responses e.g. in json to test the parsers. Also helps us check if the json conforms to the response types that we defined. I know some of the other parser tests use JS objects, but I think the test is more useful if it uses the raw response. I'm up for further discussion if we don't agree yet.

Reply:
➕ I agree with this.